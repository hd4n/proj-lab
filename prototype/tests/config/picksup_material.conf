#item_nev id
#equipment, material
@items
am a1

#mezo_nev id [tartalom]
@fields
wa w1 a1

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 w1 null null null