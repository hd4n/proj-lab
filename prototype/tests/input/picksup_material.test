load picksup_material.conf
interact v1 collect_material
getstats v1
getstats w1
