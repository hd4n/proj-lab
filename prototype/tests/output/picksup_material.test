Load: sikeres
interact: v1, mezo hash: 146611050
stats: v1 hash: 1722023916
citizens.Virologist@66a3ffec
stats: w1 hash: 146611050
map.Warehouse@8bd1b6a
