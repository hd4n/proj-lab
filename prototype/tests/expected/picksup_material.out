Load: sikeres

stats: v1
	fi_w1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
interact: v1 w1

stats: v1
	fi_w1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_a1
	ag_null
	eq_null
	eqa_null
	ef_null

stats: w1
	ci_v1
	ne_null
	eq_null
